ui/src/components/flows/tasks/TaskObject.vue

```diff
@@ -40,6 +40,7 @@
             :root="root"
             :schema="schema"
             :definitions="definitions"
+            @update:model-value="onInput"
```

Comment:
The `onInput` method has been removed, are you sure it's correct here?

Reply:
Yes because we use the mixins Task that already have the `onInput` method which is the good one for this component, the  `onInput` on this component was overriding the other one.